<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="12" column="0" colspan="13">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="btnPlay">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="8" column="12">
     <widget class="QPushButton" name="btnCreate">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="btnAddWav">
      <property name="text">
       <string>Add File</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="btnStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="btnRemove">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QLabel" name="lblSliceCOunt">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" rowspan="2" colspan="5">
     <widget class="QGroupBox" name="groupAttr">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>OT Attributes</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="dropLoop"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblGain">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Gain: 0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="sliderGain">
         <property name="minimum">
          <number>-48</number>
         </property>
         <property name="maximum">
          <number>48</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblBPM">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>BPM: 125</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="sliderBPM">
         <property name="minimum">
          <number>120</number>
         </property>
         <property name="maximum">
          <number>1200</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
         <property name="sliderPosition">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="dropStretch"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QComboBox" name="dropQuant"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="8" colspan="7">
     <widget class="QListWidget" name="listSlices"/>
    </item>
    <item row="2" column="7">
     <widget class="QGroupBox" name="groupSliceMode">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>90</height>
       </size>
      </property>
      <property name="title">
       <string>Slice Mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="radioSliceNormal">
         <property name="text">
          <string>Normal</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioSliceGrid">
         <property name="text">
          <string>Grid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioSliceSteps">
         <property name="text">
          <string>Steps</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QGroupBox" name="groupBitDepth">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Bit Depth</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radio16">
         <property name="text">
          <string>16</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio24">
         <property name="text">
          <string>24</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QGroupBox" name="groupChannels">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Channels</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioMono">
         <property name="text">
          <string>Mono</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioStereo">
         <property name="text">
          <string>Stereo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="12">
     <widget class="QGroupBox" name="groupSampleRate">
      <property name="minimumSize">
       <size>
        <width>82</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Samplerate</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radio22">
         <property name="text">
          <string>22050</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio32">
         <property name="text">
          <string>32000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio44">
         <property name="text">
          <string>44100</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio48">
         <property name="text">
          <string>48000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="8" colspan="4">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="btnAddSilence">
        <property name="text">
         <string>Add Silence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dropSlicePerSteps"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="openProject"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuOPTIONS">
    <property name="title">
     <string>OPTIONS</string>
    </property>
    <addaction name="actionSet_projects_default_path"/>
    <addaction name="menuAudioPath"/>
    <addaction name="actionSet_output_default_path"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuOPTIONS"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="openProject">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSet_projects_default_path">
   <property name="text">
    <string>Set projects default path</string>
   </property>
  </action>
  <action name="menuAudioPath">
   <property name="text">
    <string>Set audiosource default path</string>
   </property>
  </action>
  <action name="actionSet_output_default_path">
   <property name="text">
    <string>Set output default path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
